test/markup/cpp/function-title.expect.txt

```diff
@@ -1,10 +1,10 @@
-<span class="hljs-function"><span class="hljs-type">int</span> <span class="hljs-title">main</span><span class="hljs-params">(<span class="hljs-type">int</span> argc, <span class="hljs-type">char</span>** argv)</span> </span>{
-  <span class="hljs-keyword">auto</span> a = std::<span class="hljs-built_in">make_unique</span>&lt;A&gt;();
-  B *b = <span class="hljs-keyword">new</span> <span class="hljs-built_in">B</span>();
-  <span class="hljs-type">int</span> c = c * <span class="hljs-built_in">sum</span>(<span class="hljs-number">1</span>, <span class="hljs-number">2</span>);
-  <span class="hljs-keyword">if</span> (a-&gt;<span class="hljs-built_in">check1</span>())
+<span class="hljs-function"><span class="hljs-type">int</span> <span class="hljs-title">main</span><span class="hljs-params">(<span class="hljs-type">int</span> argc, <span class="hljs-type">char</span><span class="hljs-operator">*</span><span class="hljs-operator">*</span> argv)</span> </span>{
+  <span class="hljs-keyword">auto</span> a <span class="hljs-operator">=</span> std<span class="hljs-operator">::</span><span class="hljs-built_in">make_unique</span>&lt;A&gt;();
```

Comment:
I was trying not to match `<>` in this case with that and now I'm changing my mind and will revert that change.